src/main/java/SchedAddRemove.java

```diff
@@ -27,8 +27,11 @@ public static void add_course_by_index() {
             int idx = Integer.parseInt(GeneralUtils.input("Enter the index of the course you would like to add: "));
             if(idx < 1) throw new IllegalArgumentException("Error: index must be greater than zero.");
             if(idx > Main.search.get_filtered_results().size()) throw new IllegalArgumentException("Error: that index is too large.");
-            if(Main.currentsched.add_course(Main.search.get_filtered_results().get(idx-1)))
-                Main.afl.println(Main.search.get_filtered_results().get(idx-1).short_str(true) + " has been added to the current schedule");
+            if(Main.currentsched.add_course(Main.search.get_filtered_results().get(idx-1))) {
+                //if in wishlist, remove it from there
+                Main.currentaccnt.get_wishlist().remove(Main.search.get_filtered_results().get(idx-1));
```

Comment:
Does this removes all instances of the class? or only the one added to the schedule? (thinking the "add without course section")